<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>695</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sub3 Suite</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidgetMain">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabPassive">
       <attribute name="title">
        <string>Passive</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabWidgetPassive">
          <property name="font">
           <font>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabActive">
       <attribute name="title">
        <string>Active</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabWidgetActive">
          <property name="font">
           <font>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabEnums">
       <attribute name="title">
        <string>Enums</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabWidgetEnums"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1033</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenProject"/>
    <addaction name="actionRecentProjects"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionConfig"/>
    <addaction name="actionSaveCopy"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionlogViewer"/>
    <addaction name="actionDocumentation"/>
    <addaction name="separator"/>
    <addaction name="actionDonate"/>
    <addaction name="separator"/>
    <addaction name="actionBlog"/>
    <addaction name="actionTwitter"/>
    <addaction name="actionSourceCode"/>
    <addaction name="actionCheckUpdates"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionApiKeys"/>
    <addaction name="actionLanguage"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfig"/>
   <addaction name="menuProject"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <action name="actionOpenProject">
   <property name="icon">
    <iconset>
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionRecentProjects">
   <property name="icon">
    <iconset>
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open recent</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="icon">
    <iconset>
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset>
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionBlog">
   <property name="icon">
    <iconset>
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Blog</string>
   </property>
  </action>
  <action name="actionApiKeys">
   <property name="icon">
    <iconset>
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>API Keys</string>
   </property>
  </action>
  <action name="actionlogViewer">
   <property name="icon">
    <iconset>
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Log Viewer</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="icon">
    <iconset>
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Config</string>
   </property>
  </action>
  <action name="actionTwitter">
   <property name="icon">
    <iconset>
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Twitter</string>
   </property>
  </action>
  <action name="actionLanguage">
   <property name="text">
    <string>Language</string>
   </property>
  </action>
  <action name="actionSourceCode">
   <property name="icon">
    <iconset>
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Source Code</string>
   </property>
  </action>
  <action name="actionDonate">
   <property name="icon">
    <iconset>
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Donate</string>
   </property>
  </action>
  <action name="actionCheckUpdates">
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
  <action name="actionSaveCopy">
   <property name="text">
    <string>Save copy</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
